param_sweeps/__init__.py

```diff
@@ -5,4 +5,4 @@
 #  param-sweeps is distributed under the terms and conditions of the MIT License
 #  (see LICENSE file at the root of this source code package).
 
-__version__ = "0.1.2"
+__version__ = "0.1.4"
```

Comment:
Good call, I wouldn't have remembered to update this version!!

Reply:
I've been burned many times!